aten/src/ATen/native/quantized/cpu/qconv.cpp

```diff
@@ -74,6 +74,15 @@ class QConv2dInt8 final : public c10::OperatorKernel {
       int64_t groups,
       double output_scale,
       int64_t output_zero_point) {
+    // Quantized kernels are all written with NHWC (channels last) layout in
+    // mind. Ideally, we'd be compatible with conv2d behavior and preserve the
+    // inputs layout as is (doing necessary upconversions).
+    //
+    // However, to be more robust, we'd just force output layout to always be
```

Comment:
"we'd just..." doesn't make it clear whether this is something you're proposing as a follow-up, or if this is how it's currently implemented